Moved Neale while I was in there. Added Ovito and VMD to WIKI (#42)
cmelab/getting-started: wiki/pages/How_to_visualize.md

> #What does it mean to visualize a simulation?

Reviewer: ```suggestion
# What does it mean to visualize a simulation?
```
I think we need a space here

An advantage of simulating materials is that we can learn, not only about 
how a material will settle given, for example, the number of particles, density, and temperature,
but also how the system evolved at every step from beginning to end. With every atom's position tracked
throughout the course of the simulation, the first investigation into the nature of the system 

> is often to the play the "movie" of where the atoms went and who they fraternized with along the way.

Reviewer: love this description :)

## Tools for the job.

The cmelab is currently utilizing two distinct resources for this purpose. They both have their pros/cons. 

> 1. [VMD](http://www.ks.uiuc.edu/Research/vmd/) is an open source software package. The interface may be unintuitive to begin, but our lab has more expertise with this tool at present. 

Reviewer: nice, let's mention the vmdrc file included in the config_files directory of this repo

Reviewer: we might also want to mention that more detailed install instructions for vmd and the gsd plugin are in `pages/Setting_up_an_OSX_Computer.md` and `pages/Installing_the_Frequently-Used_Programs_for_CME-Lab.md`

# What does it mean to visualize a simulation?

An advantage of simulating materials is that we can learn, not only about 
how a material will settle given, for example, the number of particles, density, and temperature,
but also how the system evolved at every step from beginning to end. With every atom's position tracked
throughout the course of the simulation, the first investigation into the nature of the system 
is often to the play the "movie" of where the atoms went and who they fraternized with along the way.

## Tools for the job.

The cmelab is currently utilizing two distinct resources for this purpose. They both have their pros/cons. 

1. [VMD](http://www.ks.uiuc.edu/Research/vmd/) is an open source software package. The interface may be unintuitive to begin, but our lab has more expertise with this tool at present. VMD is highly configurable! An example .vmdrc file

> for doing so can be found [here](https://github.com/cmelab/getting-started/tree/master/config_files). This file tells

Reviewer: ```suggestion
for doing so can be found [here](../config_files). This file tells
```
I think we can link this like this--which'll make it stay on the same branch

> VMD how to look and react to a specific user. You can create or edit this file by typing ~/.vmdrc in your 
> command line. VMD requires a plugin to read the output from most of the simulations run in this lab (gsd files).

Reviewer: ```suggestion
VMD how to look and react to a specific user. You can get this file locally by running `make` in the root directory of this repository (this'll copy all the config files into your home directory) or by copying the files individually. Once copied, you can view or edit this file by running `vim ~/.vmdrc` in your 
command line. VMD requires a plugin to read the output from most of the simulations run in this lab (gsd files).
```
suggestion to make it a little more clear? also use backtick to format code examples.

> Installation instructions can be found [here](https://github.com/cmelab/getting-started/blob/master/wiki/pages/Installing_the_Frequently-Used_Programs_for_CME-Lab.md) under the "Installing Visual Molecular Dynamics (VMD)" header.

Reviewer: ```suggestion
Installation instructions can be found [here](pages/Installing_the_Frequently-Used_Programs_for_CME-Lab.md) under the "Installing Visual Molecular Dynamics (VMD)" header.
```

> Installation instructions can be found [here](pages/Installing_the_Frequently-Used_Programs_for_CME-Lab.md) under the "Installing Visual Molecular Dynamics (VMD)" header.

Reviewer: ```suggestion
Installation instructions can be found [here](Installing_the_Frequently-Used_Programs_for_CME-Lab.md)
 under the "Installing Visual Molecular Dynamics (VMD)" header.
```
oops sorry forgot we don't need the `pages` folder
